src/simplify.rs

```diff
@@ -91,7 +92,7 @@ pub(crate) struct SimplifiedText {
 
 /// Simplify message text for chat display.
 /// Remove quotes, signatures, trailing empty lines etc.
-pub(crate) fn simplify(mut input: String, is_chat_message: bool) -> SimplifiedText {
+pub fn simplify(mut input: String, is_chat_message: bool) -> SimplifiedText {
```

Comment:
Sth, like this might be better:

```rust
#[cfg(feature = "internals")]
pub fn simplify__fuzzing(mut input: String, is_chat_message: bool) -> SimplifiedText {
    simplify(input, is_chat_message)
}

pub(crate) fn simplify(mut input: String, is_chat_message: bool) -> SimplifiedText {
```

Alternatively or additionally:
```rust
#[cfg(feature = "internals")]
pub mod simplify;
#[cfg(not(feature = "internals"))]
mod simplify;
```

Reply:
Don't you need to declare this feature in the Cargo.toml and enable the feature from the fuzzing's Cargo.toml?